app/routes/__index/certificate/information.tsx

```diff
@@ -45,10 +45,12 @@ export default function CertificateInstructionsRoute() {
           Unfortunately, there are malicious websites that do not provide any security and cannot
           prove their identity to your web browser. To ensure your safety while browsing the
           internet, modern browsers automatically block access to such sites that could harm your
-          computer, as illustrated&nbsp;
-          <Link color="brand.500" href="https://expired-rsa-dv.ssl.com">
-            here.
+          computer. For example, clicking on a dangerous link will result in a warning, as
```

Comment:
"dangerous" is probably not the right word.  Here, the certificate has expired, or is not set up correctly.  I would say, "opening an https URL to a site with an expired or misconfigured certificate will result in a warning"